src/EllCrv/MinimalModels.jl

```diff
@@ -894,29 +896,35 @@ end
 Returns the reduced global minimal model of $E$.
 """
 function minimal_model(E::EllCrv{fmpq})
-  return tates_algorithm_global(E)
+  F = laska_kraus_connell(E)
+  phi = isomorphism(E, F)
+  return F, phi, inv(phi)
 end
 
 @doc Markdown.doc"""
-    minimal_model(E::EllCrv{fmpq}, p::Int) -> EllCrv{fmpq}
+    minimal_model(E::EllCrv{fmpq}, p::Int) -> EllCrv{fmpq}, 
+      EllCrvIso{fmpq}, EllCrvIso{fmpq}
 
 Returns a model of $E$, which is minimal at $p$. It is assumed that $p$
 is prime.
 """
 function minimal_model(E::EllCrv{fmpq}, p::Int)
   Ep = tates_algorithm_local(E, p)
-  return Ep
+  phi = isomorphism(E, Ep)
+  return Ep, phi, inv(phi)
```

Comment:
```suggestion
  return Ep, phi
```
Should we do the same as for the global minimal model?

Reply:
I think it's more convenient for the user to return both the map and its inverse everywhere. That's also the way Magma does it.The global minimal model also returns both maps.

Reply:
Sorry, I did not know that the global minimal model also returned both. Then I propose to also change the global minimal model (but I will do this in a follow up PR). I think returning both maps in Hecke is from a time when there were no proper map objects and calling `inv` was not an option.

I find the Magma behavior inconsistent with the rest of Magma (but also inconsistent in the elliptic curve world itself, since `MinimalModel(E, p) -> CrvEll, Map` in the number field case).

So I'd rather have `minimal_model` in its various forms behave like all other cases where we return an invertible map, e.g. `is_isomorphic`.

@fieker Do you have an opinion?